www/partial/editor.html

```diff
@@ -15,6 +15,10 @@
 		<!--<form action="/download" method="post">-->
 		<!--<input type="hidden" name="data" ng-value="pro5Ctrl.parsed">-->
 		<button ng-click="pro5Ctrl.download()">Download {{pro5Ctrl.templateExt}} file</button>
+		<div id="checkbox-container">
+			<input ng-model="referenceCaps" ng-change="pro5Ctrl.referenceChangeCaps(referenceCaps)" type="checkbox"><span>Reference Caps</span></input><br>
```

Comment:
Please use `<label>` instead of span, and either nest the input inside the label, or use the for attribute. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label